### PR TITLE
APP-1075: claim status detail card from new status cards api

### PR DESCRIPTION
**Pull request by StylianosGakis**
### Checklist

- [ ] Functionality is covered by an integration test
- [X] Functionality is accessible in engineering mode - Showkase -> Claim Status -> ClaimStatusCard

|showkase|app|app-dark|
|---|---|---|
|image|image|image|

### CODE REVIEW

File: app/src/main/java/com/hedvig/app/feature/home/ui/claimstatus/composables/ClaimPills.kt [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        PillData.PillType.OPEN -> OutlinedPill(text)
+        PillData.PillType.CLOSED -> Pill(text, MaterialTheme.colors.primary)
+        PillData.PillType.REOPENED -> Pill(text, ClaimStatusColors.Pill.reopened)
+        PillData.PillType.PAYMENT -> Pill(text, ClaimStatusColors.Pill.paid)
+        PillData.PillType.UNKNOWN -> OutlinedPill(text)
+    }
+}
+
+@Preview
+@Preview(uiMode = UI_MODE_NIGHT_YES)
```
Comment (hedvigoscar) [inline app/src/main/java/com/hedvig/app/feature/home/ui/claimstatus/composables/ClaimPills.kt:60]: Does this create two `showkase`-previews? 🤔 Also, should it be grouped?

**Review by hedvigoscar — APPROVED** [review]
LGTM!

File: app/src/main/java/com/hedvig/app/feature/home/ui/claimstatus/composables/ClaimPills.kt (hunk 1 above)
Comment (StylianosGakis) [reply app/src/main/java/com/hedvig/app/feature/home/ui/claimstatus/composables/ClaimPills.kt:60]: This creates *no* showkase previews actually https://github.com/airbnb/Showkase/issues/171.
Which is precisely why https://github.com/HedvigInsurance/android/pull/808/files#diff-d010a954ceb4addb5e1128f4097feeead5a18bc07e2f65ddfe08e41eedc69f8eR62-R79 exists which puts the entire card in showkase with the appropriate grouping etc.
Is this fine? Any other suggestions/proposals if not?

Comment (hedvigoscar) [reply app/src/main/java/com/hedvig/app/feature/home/ui/claimstatus/composables/ClaimPills.kt:60]: Yes, it's all good

**Pull request merged**